script.js

```diff
@@ -25,3 +25,206 @@ const LiInMenu = document.getElementsByClassName('menue');
+    switch (i) {
+      case 2:
+        dynmicTitle = 'Data Dashbord HealthCare';
+        dynamicImage = " <img src='images/availiability.png'>";
+        break;
+      case 3:
+        dynmicTitle = 'Website portfolio';
+        dynamicImage = " <img src='images/availiability.png'>";
+        break;
+      case 4:
+        dynmicTitle = 'Professional Art Printing Data More';
+        dynamicImage = " <img src='images/availiability.png'>";
+        break;
+      case 5:
+        dynmicTitle = 'Data Dashbord HealthCare';
+        dynamicImage = " <img src='images/availiability.png'>";
+        break;
+      case 6:
+        dynmicTitle = 'Professional Art Printing Data More';
+        dynamicImage = "<img src='images/availiability.png'>";
+        break;
+      case 0:
+        dynmicTitle = 'Multi-post Stories';
+        dynamicImage = "<img src='images/woman.png'>";
+        break;
+      case 1:
+        dynmicTitle = 'Professional Art Printing Printing Data';
+        dynamicImage = '   <img src="images/professional.png">';
+        break;
+      default:
+        dynamicImage = 'no image';
+        dynmicTitle = 'no title';
+    }
+    const divParent = document.createElement('div');
+    const popUp = document.createElement('div');
+    popUp.className = 'popUpParent';
+    popUp.innerHTML = `
+   <div class='popUpHeader'>
+   <h1>${dynmicTitle}</h1>
+   <i class='bx bx-x'></i>
+   </div>
+   <ul>
+   <li>HTML</li>
+   <li>Bootstrap</li>
+   <li>Ruby on Rails</li>
+   </ul>
+${dynamicImage}
+   <p>             Lorem ipsum dolor sit amet
+    consectetur adipisicing elit.
+     Similique cumque fugit, corrupti odio 
+     ex eos provident reprehenderit expedita
+      exercitationem saepe dolor.
+      e  ex eos provident reprehenderit expedita
+      exercitationem saepe dolor.
+      ex eos provident reprehenderit expedita
+      exercitationem saepe dolor.
+       consectetur adipisicing elit. Quos illum ex ducimus quaerat
+        inventore officia blanditiis impedit suscipit,
+         unde nulla, porro magnam ex eos provident reprehenderit expedita
+         exercitationem saepe dolor.x eos provident reprehenderit expedita
+      exercitationem saepe dolor.
+       consectetur adipisicing elit. Quos illum ex ducimus quaerat
+        inventore officia blanditiis impedit suscipit,
+         unde nulla, porro magnam ex eos provident reprehenderit expedita
+         exercitationem saepe dolor.
+         
+          consectetur adipisicing elit. Quos illum ex ducimus quaerat
+           inventore officia blanditiis impedit suscipit,
+            unde nulla, porro magnam
+       consectetur adipisicing elit. Quos illum ex ducimus quaerat
+        inventore officia blanditiis impedit suscipit,
+         unde nulla, porro magnam
+          incidunt corporis quod 
+         repellat mollitia eum nesciunt! Sunt!</p>
+   <div class='popUpFooter'>
+   <button>See live</button>
+   <button>See source </button>
+   </div>
+  `;
```

Comment:
- [ ] kindly consider sticking to the given design as much as possible  (e.g., font, colors, images, text, margins) here am emphasizing margins.to make your website an. exact duplicate of the provided Figma either on mobile popup or desktop popup check the images below to see what needs to be fixed
- your mobile popup design 👇 
<img width="371" alt="Screen Shot 2023-06-29 at 14 03 49" src="https://github.com/ZabihullahNooriWardak/Zabih-portfolio/assets/60146030/9a0a2b9d-dbfb-48ad-9a59-533fe6e126cc">

- your desktop pop-up design 👇 
<img width="1413" alt="Screen Shot 2023-06-29 at 14 00 46" src="https://github.com/ZabihullahNooriWardak/Zabih-portfolio/assets/60146030/79a665f0-4925-45bd-840c-0f31db02e29b">

- given mobile popup design 👇 
<img width="286" alt="Screen Shot 2023-06-29 at 14 18 29" src="https://github.com/ZabihullahNooriWardak/Zabih-portfolio/assets/60146030/afb01e7a-53b0-4f22-a5a3-8eb13ba7207b">

- given desktop pop-up design 👇 
<img width="822" alt="Screen Shot 2023-06-29 at 14 19 55" src="https://github.com/ZabihullahNooriWardak/Zabih-portfolio/assets/60146030/8e2f7f7d-fdd3-428a-af67-cdbffa012699">